### ISSUE TITLE
Different returning type values

### ISSUE THREAD

**Issue by jungla88** [AdeelH/pytorch-multi-class-focal-loss#6]
Hi there!

I think a little line of the code can cause ValueError or something especially when used in automatic optimization like Lightning.
In particular, line 69 should return a torch.Tensor type and not a float, as your (correctly) type annotation of forward function.

**Comment by AdeelH** [comment]
Good catch! Thanks! I think `return torch.tensor(0.)` is what we want.

I'd be happy to take a PR, if you are interested.
